### PR TITLE
polymer++

### PR DESCRIPTION
**Pull request by notwaldorf**
`x-style` -> `custom-style`
`mixin` -> `@apply`
`version++`
`polymer++`

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -19,13 +19,13 @@
...
   "ignore": [],
   "dependencies": {
-    "polymer": "Polymer/polymer#v0.8.0-rc.7",
-    "iron-flex-layout": "PolymerElements/iron-flex-layout#^0.8.0"
+    "polymer": "Polymer/polymer#v0.9.0-rc.1",
+    "iron-flex-layout": "PolymerElements/iron-flex-layout#^0.9.0"
   },
   "devDependencies": {
     "webcomponentsjs": "webcomponents/webcomponentsjs#^0.6.1",
     "web-component-tester": "Polymer/web-component-tester#^2.2.3",
```
Comment (cdata) [inline bower.json]: Should be `"*"`

**Comment by notwaldorf** [comment]
Done!

File: paper-spinner.html [hunk 2]
```diff
@@ -54,7 +54,7 @@
 -->
 
 <dom-module id="paper-spinner">
-  <style is="x-style">
+  <style is="custom-style">
   * {
     --paper-spinner-layer-1-color: #4285f4;
     --paper-spinner-layer-2-color: #db4437;
```
Comment (cdata) [inline paper-spinner.html]: Inline defaults por favor.

**Comment by cdata** [comment]
:+1:

**Pull request merged**